samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/EvaluationMessages_eu.properties

```diff
@@ -339,3 +339,6 @@ objPercent=Ehuneko zuzena
 # S2U-15
 submission_nav_next_submission=Hurrengora
 submission_nav_previous_submission=Aurrekora
+
+# S2U-5
+unsavedchangesrubric=Hay cambios sin guardar en los criterios. Debe confirmar o cancelarlos
```

Comment:
Hi @PabloSanRoman, could you help with this translation please?

Reply:
unsavedchangesrubric=Gorde gabeko aldaketak daude irizpideetan. Berretsi edo utzi